### DISCUSSION TITLE
Hero Session 5 as poor mans dashcam?

### DISCUSSION THREAD

**Discussion by DAWGSofW4R** [gopro/labs#559] [Q&A]
I know the Session 5 is old as hell, I think the latest version of my firmware is 2.51.74 but it looks like what I'm trying to accomplish is still supported, I'm just having issues making it work.

Use a QR of !MWAKE=2mVLr1080p30fW

The idea being I'll set video capture to a 20 minute loop natively on the camera, and then use the QR code to wake the camera when USB power is applied, and record at 1080P/30FPS in a loop length determined by what I've set the camera to.

Unfortunately I have no idea what I'm doing. I'm not sure if the !WAKE command is in the wrong part of the string, or how to set up a boot script so that it starts recording as soon as it wakes. Does that script need to be in a .txt file saved to the SD card or is it part of the QR? Also no idea how to get it to STOP recording when USB power is disconnected.

I tried using the USB Power Trigger option on the desktop version with limited success, and was told the WAKE=2 method was far more reliable.

**Comment by dnewman-gpsw** [comment]
The command look okay for a more recent camera, yet WAKE is not supported on H5S (I just check the code, that feature came later.)  Even the old USB Power Trigger is not listed as supported.  I can't think I any solution using that model.

**Reply by DAWGSofW4R** [reply]
I could get the USB Power Trigger to work, but it was very inconsistent... some tests it would work the first time after I plugged it in, some other times I could get it to work 2-3 times in a row, but it always stopped. Just wanted to make sure I was on the right track with the code I was using to create the QR using the WAKE command before I gave up lol

**Comment by robertken** [comment]
Not sure about the looping aspect, but this seems to work for me on my session 5 running 2.51.72.

QR of !uN!S!u10E!R  I generated QR here: [](https://gopro.github.io/labs/control/usb/).

This will record immediately after USB power is applied, with whatever the standard recording settings are, and stop recording 10 seconds after USB power is removed. This behavior will repeat for multiple USB power cycles. 

However, this functionality breaks, and a QR needs to be re-scanned if you use the camera by pressing the physical record button.

Maybe @dnewman-gpsw can comment if the H5S stores the commands in a file to attempt to maintain the QR'd functionality after pressing the record button.
